src/components/AccessibilityGuide/StickyElements.mdx

```diff
@@ -0,0 +1,77 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Accessibility Guide/Sticky Elements" />
+
+# Sticky Elements
+
+## Table of Contents
+
+- {<Link href="#general-information" target="_self">General Information</Link>}
+- {<Link href="#implementation" target="_self">Implementation</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## General Information
+
+**Edwin/Marty: Why do we use sticky headers or sticky elements in general?**
+
+Sighted keyboard users tab through a page to display more content and to access buttons and links.
```

Comment:
While waiting for the context in "General Information", there can be a new section titled "General Issues". You can start off by saying that if you absolutely must use sticky elements on a page, that you need to think about UX issues such as taking up space, breaking when zooming in on a mobile device, difficult to access when tabbing forward and backwards, and hiding focused elements (examples taken from here https://adamsilver.io/blog/the-problem-with-sticky-menus-and-what-to-do-instead/)

Then you can state the problem faced in the new DC homepage and then keep everything after "Sighted keyboard users....". It can be clear that you're focusing only on the sticky header hiding focused elements issue and not covering the other issues.